### ISSUE TITLE
Support npm module oortcloud/node-ddp-client

### ISSUE THREAD

**Issue by sakukode** [eclipsesource/tabris-js#467]
https://github.com/oortcloud/node-ddp-client

can you help to support this npm module? thanks..

**Comment by radiegtya** [comment]
Yeah I need this too, Meteor is awesome FW out there. I think Tabris should support this ASAP. I got error when installing npm module which is having module dependencies. For example:

MyApps
-node_modules
---underscore (this npm module works fine, because it has no node_modules dependencies)
---ddp (this npm module not working because it has node_modules dependencies)
-----node_modules (node_modules by the ddp npm module)

Please answer, thanks

**Comment by jonek** [comment]
We will look into this for the next 1.0.1 release.
It would also help if you could provide a clear description of the problem (i.e. warnings/errors on the console). It would also help if you could provide clear steps to reproduce the problem.

**Comment by jonek** [comment]
A more specific error description can be found in #470.

**Comment by ralfstx** [comment]
This module requires the built-in node module [events](https://nodejs.org/docs/latest/api/events.html) that is not yet available for Tabris.js.

**Comment by radiegtya** [comment]
So this will be supported or not?

**Comment by ralfstx** [comment]
The ddp module also has a dependency to a websocket implementation, that will also not work out of the box, as it in turn depends on the node net module.

Chances are that it will be supported at some point, but it won't happen in the next weeks.

**Comment by sand123** [comment]
Hi!
So, fetching from server via http ajax is the only option for now?

**Comment by radiegtya** [comment]
Hope it will be realized ASAP. As I said before, meteor is very good, promising, and have very high github stars. Automatically if tabris able to support this, tabris name will be gained popularity more and more.
